src/Savegame/SavedBattleGame.cpp

```diff
@@ -58,7 +58,7 @@ namespace OpenXcom
 SavedBattleGame::SavedBattleGame(Mod *rule, Language *lang) :
 	_battleState(0), _rule(rule), _mapsize_x(0), _mapsize_y(0), _mapsize_z(0), _selectedUnit(0),
 	_lastSelectedUnit(0), _pathfinding(0), _tileEngine(0), _enviroEffects(nullptr), _ecEnabledFriendly(false), _ecEnabledHostile(false), _ecEnabledNeutral(false),
-	_globalShade(0), _side(FACTION_PLAYER), _turn(0), _bughuntMinTurn(20), _animFrame(0),
+	_globalShade(0), _side(FACTION_PLAYER), _turn(0), _bughuntMinTurn(20), _animFrame(0), _nameDisplay(false),
```

Comment:
_nameDisplay(true) ?

Reply:
I thought the code would still adhere to your guidelines with _nameDisplay(false) since it won't do anything at all if no corresponding ruleset is active. If that assumption is wrong, I can change it.

Reply:
it's ok then